interface/weno/RxLog.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * weno rxlog.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+
+require_once("../globals.php");
+use OpenEMR\Core\Header;
+
+?>
+<h1><?php print xlt("Rx Log"); ?></h1>
+<html>
+<head>
```

Comment:
place this here (this will then give the tab of the frame a title) below the above head tag:
<title><?php print xlt("Rx Log"); ?></title>